.bumpversion.cfg

```diff
@@ -3,9 +3,11 @@ current_version = 0.17.0-alpha
 commit = True
 tag = False
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\-[a-z]+)?
-serialize = 
+serialize =
 	{major}.{minor}.{patch}-alpha
 
 [bumpversion:file:.bumpversion.cfg]
 
 [bumpversion:file:.env]
+
+[bumpversion:file:docs/tutorials/upgrading-airbyte.md]
```

Comment:
I added a bump tag for the doc so it would always be up-to-date with the latest released version (easier to copy/paste without having to change the version number for the user)

Reply:
nice idea!